cleanlab/classification.py

```diff
@@ -355,10 +386,52 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
                 clf_kwargs=clf_kwargs,
             )
         elif self.verbose:
-            print("Using provided label_issues_mask instead of finding label issues.")
+            print("Using provided label_issues instead of finding label issues.")
 
-        # Always overwrites `self.label_issues_mask` and ensures same length as `X` and `labels`.
-        self.label_issues_mask = label_issues_mask
+        # label_issues always overwrites self.label_issues_df. Ensure it is properly formatted:
+        if isinstance(label_issues, pd.DataFrame):
```

Comment:
Maybe move all of the parameter checking logic to helper function if this code can only be used here or to utils if this code can be re-used across the package.

Reply:
added helper class method _process_label_issues_arg